.circleci/config.yml

```diff
@@ -1,5 +1,5 @@
 version: 2.1
```

Comment:
I would maybe expect to remove whole `.circleci` folder and not to do changes if we want to migrate anyway WDYT?